### PR TITLE
Fix css

### PR DESCRIPTION
**Pull request by ketankartoza**
This PR is for #833
Before:-
image

After:-
image

### CODE REVIEW

File: django_project/frontend/src/containers/MainPage/Metrics/index.scss [hunk 1]
```diff
@@ -22,10 +22,11 @@
 
     .white-chart {
         background: #FFF;
-        box-shadow: 0px 4px 4px 0px rgba(0, 0, 0, 0.25);
+        // box-shadow: 0px 4px 4px 0px rgba(0, 0, 0, 0.25);
```
Comment (dimasciput) [inline django_project/frontend/src/containers/MainPage/Metrics/index.scss:25]: Please just remove this

**Review by dimasciput — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 25 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 25 min

**Review by dimasciput — APPROVED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 26 min

**Pull request merged**